src/_sass/_related.scss

```diff
@@ -40,10 +40,60 @@
     display: flex;
   }
 
-  .nested-navigation {
+  .related__section-link {
+    position: relative;
+    label,
+    a {
+      display: inline;
+    }
+  }
+
+  .related__section-list {
+    display: none;
     margin-left: 1em;
   }
 
+  .related__expand-toggle {
+    -webkit-appearance: none;
+    appearance: none;
+    background-color: transparent;
+    margin: 0;
+  }
```

Comment:
Might be worth adding some `margin-left` on small screens here. The menu is a bit more usable on small screens with the toggles, but they're very close to the text